client_test.go

```diff
@@ -628,6 +628,93 @@ func TestClientRemoveRelation(t *testing.T) {
 	}
 }
 
+func TestClientModifyRelation(t *testing.T) {
+	c := qt.New(t)
```

Comment:
Minor: could we theoretically run this test in parallel, i.e. `t.Parallel()`? Sub-tests also are not run in separate goroutines.

Or is it the restriction of the HTTP server?

Reply:
It is indeed a limitation of the http mock server. Since we use the mock server to verify that the requests are as expected, we can't hit the server simultaneously with different tests.